test_responses.py

```diff
@@ -127,6 +157,46 @@ def run():
     assert_reset()
 
 
+@pytest.mark.parametrize(
+    "original,replacement",
+    [
+        (
+            Response(
+                method=responses.GET,
+                url="http://example.com/api",
+                body="original",
+                status=202,
+            ),
+            Response(
+                method=responses.GET,
+                url="http://example.com/api",
+                body="replacement",
+                status=299,
+                reason="warning",
+            ),
+        ),
+    ],
+)
+def test_replace_with_reason(original, replacement):
```

Comment:
I think we don't need this test.

Reply:
Replace must be tested separately I think. The given `test_replace` adds more responses than necessary so I'm not sure what is its purpose.

Reply:
Replace must be tested separately I think. The given test_replace adds more responses than necessary so I'm not sure what is its purpose.